src/MDMMotionAnimator.m

```diff
@@ -237,11 +237,11 @@ - (void)addAnimation:(CABasicAnimation *)animation
 
   NSString *key = animation.additive ? nil : keyPath;
 
-  MDMConfigureAnimation(animation, timing);
+  MDMConfigureAnimation(animation, traits);
 
-  if (timing.delay != 0) {
+  if (traits.delay != 0) {
     animation.beginTime = ([layer convertTime:CACurrentMediaTime() fromLayer:nil]
-                           + timing.delay * timeScaleFactor);
+                           + traits.delay * timeScaleFactor);
```

Comment:
Super-nit: Since we're changing this line, I've been told that plusses should trail the preceding line, not lead the subsequent line.